shiptest.dme

```diff
@@ -3196,16 +3196,17 @@ g// DM Environment file for shiptest.dme.
 #include "voidcrew\code\vc_world.dm"
 #include "voidcrew\code\__DEFINES\cloning_defines.dm"
 #include "voidcrew\code\__DEFINES\defines.dm"
+#include "voidcrew\code\__DEFINES\is_helpers.dm"
```

Comment:
I don't think you understood what i said, this needs to be at the TOP of the file, because otherwise the code won't compile properly.